packages/containers/src/Breadcrumbs/Breadcrumbs.connect.js

```diff
@@ -9,10 +9,12 @@ export function ContainerBreadcrumbs(props) {
 	const state = props.state || DEFAULT_STATE;
 	const items = state.get('items', props.items);
 	const newProps = {
+		id: state.get('id', props.id),
```

Comment:
this state.get looks strange to me. the state is supposed to change ?

Reply:
I agree, why do we have id in state ? Shouldn't it be only the prop ?